core-container/entry.sh

```diff
@@ -309,11 +303,11 @@ fi
 if [ "$PROVIDER_SECRET_NAMESPACE" == "" ]; then
     PROVIDER_SECRET_NAMESPACE=psecret_namespace
 fi
-if [ "$SPACE_MANAGER_KUBECONFIG" == "" ]; then
+if [ "$SM_CONFIG" == "" ]; then
```

Comment:
Why is this conditional? As far as I can tell, `$SM_CONFIG` is guaranteed to be empty at this point.